src/components/ha-web-rtc-player.ts

```diff
@@ -83,7 +83,13 @@ class HaWebRtcPlayer extends LitElement {
   private async _startWebRtc(): Promise<void> {
     this._error = undefined;
 
-    const peerConnection = new RTCPeerConnection();
+    const peerConnection = new RTCPeerConnection({
+      iceServers: [
+        {
+          urls: ["stun:stun.l.google.com:19302"],
```

Comment:
We shouldn't send traffic to google without warning users about it and getting consent.